agent/engine/docker_client_test.go

```diff
@@ -363,28 +362,19 @@ func TestCreateContainerTimeout(t *testing.T) {
 	defer done()
 
 	warp := make(chan time.Time)
```

Comment:
Is `warp` used/effective anymore? Can we get rid of that as well (here and in other methods)?

Reply:
It's used to make the go routine not stop so that we don't have to handle the logic after the mocked function return. I will remove the warp add return value for the mocked function instead.